cmd/docker.lima

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+set -eu
+: "${LIMA_INSTANCE:=docker}"
+: "${LIMA_DOCKER:=docker}"
+
+DOCKER=$(command -v "$LIMA_DOCKER" || true)
+if [ -n "$DOCKER" ]; then
+  context="lima" # TODO: multiple contexts
+  exec "$DOCKER" --context=$context "$@"
```

Comment:
Instead of using docker contexts you could specify `DOCKER_HOST` explicitly:

```shell

: "${LIMA_HOME:=$HOME/.lima}"

[...]

if [ -n "$DOCKER" ]; then
  export DOCKER_HOST="unix://$LIMA_HOME/$LIMA_INSTANCE/sock/docker.sock"
  exec "$DOCKER" "$@"
else

[...]
```

That way you can use multiple instances, and you don't need to create contexts at all; you just need to set `LIMA_INSTANCE`.

Reply:
It was like that, before...

Reply:
This was just a suggestion to your `# TODO: multiple contexts` comment, and I thought if you have to specify a context/instance anyways, then you don't really need to create a context.

This is just for `docker.lima`. I would still want to create a docker context because I normally don't use multiple docker sockets simultaneously, and I want to use `docker` instead of `docker.lima`.

Note that my suggestion above does **not** require setting any environment variables for the default case; it should work out-of-the-box like:

```console
limactl start template://docker
docker.lima run -it busybox
```

Reply:
Env var SGTM